ci/pipeline

```diff
@@ -199,14 +202,14 @@ def run(publish: Boolean = false): (String, Option[awsClient.Artifact]) = {
 @main
 def jenkins(): Unit = {
   if(utils.isPullRequest) {
```

Comment:
What do you think about calling 2 distinct `@main` methods from here where each one of those should then call it's own targets? E.g. PR's don't publish artifacts and master builds do etc.

Reply:
we are likely to add another distinct `@main` as well... I'm expecting release to be it's own path.

Reply:
Done.